### PR TITLE
Left nav hot fix

### PR DESCRIPTION
**Pull request by jjmulenex**
I made a few styling changes to this branch in support of Willies work.

### CODE REVIEW

File: src/components/LeftNav.js [hunk 1]
```diff
@@ -142,7 +142,8 @@ class LeftNav extends React.PureComponent {
             onLeftIconButtonTouchTap={this.close.bind(this)}
 
             iconElementRight = {this.state.appBarIcon}
-            onRightIconButtonTouchTap = {this.state.appBarOnTouch} />
+            onRightIconButtonTouchTap = {this.state.appBarOnTouch}
+            className={'left-drawer-titleBar'} />
```
Comment (bartvde) [inline src/components/LeftNav.js]: this can be just string without the { }

File: __tests__/components/LayerList.test.js [hunk 2]
```diff
@@ -64,6 +64,20 @@ describe('LayerList', function() {
...
+  it('generates a drawer than can close', function() {
+    var container = document.createElement('div');
+    var drawerOpen = true;
+    ReactDOM.render((
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
+        <LayerList intl={intl} map={map} open={drawerOpen}/>
+      </MuiThemeProvider>
+    ), container);
+    var button = container.querySelectorAll('.layerlistbutton');
+    Simulate.click(button);
```
Comment (bartvde) [inline __tests__/components/LayerList.test.js]: in other unit tests we use the React test utils for this see e.g. https://github.com/boundlessgeo/sdk/blob/79f0c232689a7f7b643db01af3c62924006fccac/__tests__/components/Navigation.test.js#L19

File: src/components/StyleModal.js [hunk 3]
```diff
@@ -323,7 +323,9 @@ class StyleModal extends React.PureComponent {
         <AppBar
           title={formatMessage(messages.title, {layer: this.props.layer.get('title')})}
           iconElementLeft={<IconButton label={formatMessage(messages.closebutton)} > <NavigationArrowBack/> </IconButton>}
-          onLeftIconButtonTouchTap={this.close.bind(this)}/>
+          onLeftIconButtonTouchTap={this.close.bind(this)}
+          className={'left-drawer-titleBar'}
+          />
```
Comment (bartvde) [inline src/components/StyleModal.js]: same here can be just string without { }

**Comment by bartvde** [comment]
put a few small remarks in, otherwise looks good

needs to be rebased on master though

File: __tests__/components/LayerList.test.js (hunk 2 above)
Comment (wnordmann) [inline __tests__/components/LayerList.test.js]: thanks, this wasn't working that well anyway
I'll adjust

**Pull request merged**